src/engine/framework/System.cpp

```diff
@@ -549,7 +549,7 @@ static void Init(int argc, char** argv)
 #endif
 
 	// Print a banner and a copy of the command-line arguments
-	Log::Notice(Q3_VERSION " " PLATFORM_STRING " " ARCH_STRING " " __DATE__);
```

Comment:
Can't you fix that with compiler flags or something?

Reply:
With compiler flags? I do not see how. Maybe it could be made a build option though. TBH this commit is not the most interesting to me, so I would even be fine for a removal.